### ISSUE TITLE
Airplane wizzard in companion v 2.2 does not clearly indicate wing servo positions.

### ISSUE THREAD

**Issue by mrpraline** [opentx/opentx#5358]
In the airplane wizzard in the transmitter it is exactly shown which servo on the wing (left/right) is to be used for which  channel in the Aileron, Flaps, Airbrakes and Elevon screens. In Companion it just says First/Second Aileron/Flap...etc without indicating the left or right wing. Moreover the picture is turned 90 deg anticlockwise so that the top to bottom order of the servos in the wing is the reverse of the top-bottom order in the dialog. 

I suggest rewording First/Second to Left/Right and/or clearly indicating the servo positions in the pictures of the wizzard. .

**Comment by kilrah** [comment]
It's like this precisely because it cannot know/doesn't matter.

The wizard only sets up the mixer, you still have to configure the outputs page (which is the interface to the mechanical stuff) yourself afterwards on the live radio and model, and one of the things you'll do there is set servo reverse correctly according to your connections.
